src/library/dao/playlistdao.h

```diff
@@ -144,6 +144,7 @@ class PlaylistDAO : public QObject, public virtual DAO {
     QMultiHash<TrackId, int> m_playlistsTrackIsIn;
     AutoDJProcessor* m_pAutoDJProcessor;
     DISALLOW_COPY_AND_ASSIGN(PlaylistDAO);
+    void createTableIDList(QList<int>* pIdList, int playListID);
```

Comment:
OK, I do it, I didn't do it because I thought on sending the pointer to the QList, it would take less time to write the list, I do the correction